src/javascripts/modules/ExpandListItem.js

```diff
@@ -0,0 +1,26 @@
+import { debounce } from 'utility/debounce'
+
+export default class ExpandListItem {
+  constructor(el) {
+    this.el = el
```

Comment:
the way i'd do the selectors for the functionality is by putting a single `data-module` attribute here on the ul and inferring the buttons/list items within the scripts for that. it looks like you are already having to go up instead of down in the ancestor tree by defining `this.tag` on the following line.

the way i look at it is the selector you hook the functionality to should be the top level selector. nothing should need to be referenced parent-wise and as an added bonus, siblings should not be referenced either. rather it should all be children of the selected element.

i'd recommend rewriting this so that you are targeting the top level `ul` and rename it as `ExpandList`, this way you aren't having to rewrite `data-module="[selector]"` over and over in your html.

once you have done that, you can use [`Array.from()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) to create an array of node element `<li>` hook an event listener onto for a button click using [`forEach()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)

Reply:
What Ben said...also throwing in some other ways to turn NodeLists into arrays so you can use `forEach`:

`Array.prototype.slice.call(document.querySelectorAll("div"));`

or Greg's fancy way:

`[...document.querySelectorAll("div")]`

Gregs fancy way requires adding the spread operator for arrays to babel...so it takes more setting up